helpdesk_mgmt_timesheet_report_order/report/helpdesk_ticket_report_view.xml

```diff
@@ -0,0 +1,10 @@
+<odoo>
+    <template id="report_timesheet_custom_order" inherit_id="hr_timesheet.report_timesheet">
+    
+        <xpath expr="//tbody/tr[1]" position="before">
+            <t t-if="user_id.has_group('helpdesk_mgmt_timesheet_report_order.group_helpdesk_timesheet_priority_order')">
+                <t t-set="docs" t-value="docs.sorted(key=lambda r: (r.ticket_id.priority, r.ticket_id.sequence, r.ticket_id.number), reverse=True)"/>
```

Comment:
The current sorting for tickets is only in descendant mode for `priority`. With this change, is `desc` also applied for `sequence` and `number`?

Reply:
Apparently the reverse only affects the first field of the lambda, the only solution I see is to order it manually from a function, what do you think?

Reply:
Ok, then I'll test it, beacause that should be the desired behavior, al least in this case :)

Reply:
Tested, works as expected